### PR TITLE
FractionTokens

### PR DESCRIPTION
**Pull request by SMITH1103**
**Description**:
-- Fixed 'fractional token' issue
-- Fixed input issue with leading 0s in the decimal preventing smaller token transfers
-- Removed token info queries to prevent invisible transactions

### CODE REVIEW

**Review by jackthta — COMMENTED** [review]
It would be good to add validation for transferring 0ℏ

Screen Shot 2021-09-29 at 13 52 13

File: src/components/interface/AssetInput.vue [hunk 1]
```diff
@@ -65,53 +65,21 @@ export default defineComponent({
...
-        'update:modelValue',
-        inputNum?.multipliedBy(assetScale.value) ?? null,
-      )
-
-      if (assetDecimals.value > 0) {
-        nextTick(() => {
-          // IFF the input value had a trailing decimal, re-add it
-          if (inputValue.endsWith('.')) target.value += '.'
-        })
+      if(inputValue !== null && inputValue.split(".").length > 2) {
```
[the remark this answers is not in the document]
Comment (SMITH1103) [reply src/components/interface/AssetInput.vue]: Made recommended change.

**Comment by SMITH1103** [comment]
> It would be good to add validation for transferring 0ℏ
> 
> Screen Shot 2021-09-29 at 13 52 13

@jackthta Disabled adding send/saving on 0.

**Comment by jackthta** [comment]
> > It would be good to add validation for transferring 0ℏ
> > Screen Shot 2021-09-29 at 13 52 13
> 
> @jackthta Disabled adding send/saving on 0.

👍🏻 Going to wait until kabuto v2 comes back up to fully double check and test

**Pull request merged**